src/replicache.ts

```diff
@@ -264,6 +260,16 @@ export default class Replicache implements ReadTransaction {
     }
   }
 
+  private _schedulePush(interval: number | null): void {
+    // We do not want to restart the push timer.
+    if (interval && this._pushTimerId === 0) {
+      this._pushTimerId = setTimeout(() => {
+        this._pushTimerId = 0;
+        this.push();
```

Comment:
We need to makes sure that as soon as the push starts (in same turn of event loop), the push timer gets cleared, otherwise mutations will be delayed to server.

Reply:
Oh duh, it's in the very next line. nvm.

Reply:
looks right to me

Reply:
I see. let me change it to clear thing in push